### PR TITLE
Homepage, user_account and video_upload functionality proposals: goodteens

### PR DESCRIPTION
**Pull request by stankasvecova**

### CODE REVIEW

File: HOMEPAGE_README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# REDI Sprint 2020 Frontend Development 2, Final Project
```
Comment (BTBTravis) [inline HOMEPAGE_README.md]: Please make this README.md as it is the your final project concept.

File: HOMEPAGE_README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Screenshots
+
+![screen shot](homepage_proposal.png)
+
+## Features:
+
+- Registration
+- User-account: update & delete personal information
+- User-account: create, post, edit & delete an article
+- Upload a video-link
```
Comment (BTBTravis) [inline HOMEPAGE_README.md]: Perhaps this would be more clear as video upload feature and video block list / feed?

File: HOMEPAGE_README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Description:
+A user can register to the portal and access user account with following functionalities:
+Update & delete personal information
+Create, post, edit & delete an article
+Upload a video-link (not the video itself)
+
+## Screenshots
+
+![screen shot](homepage_proposal.png)
```
Comment (BTBTravis) [inline HOMEPAGE_README.md]: nice image please keep repo clean by placing this image into docs folder and deleting the existing screen shot there.

**Review by BTBTravis — CHANGES_REQUESTED** [review]
Great start! just few comments on structure of files

File: HOMEPAGE_README.md (hunk 1 above)
Comment (stankasvecova) [reply HOMEPAGE_README.md]: I didn't know how to rename it, so instead i updated the README.md that you created originally (and there is a pr for that).

File: HOMEPAGE_README.md (hunk 2 above)
Comment (stankasvecova) [reply HOMEPAGE_README.md]: reworded. thanks

File: HOMEPAGE_README.md (hunk 3 above)
Comment (stankasvecova) [reply HOMEPAGE_README.md]: there is now a pr placing the screenshot into the docs folder.

**Review by BTBTravis — APPROVED** [review]
I see you have made the requested changes good work! Approved you can merge

**Pull request merged**